library/include/flutter_desktop_embedding_core/glfw/embedder.h

```diff
@@ -66,6 +66,25 @@ FDE_EXPORT FlutterWindowRef FlutterEmbedderCreateWindow(
     int initial_width, int initial_height, const char *assets_path,
     const char *icu_data_path, const char **arguments, size_t argument_count);
 
+// Runs an instance of a headless Flutter engine.
+//
+// FlutterEmbedderInit() must be called prior to this function.
```

Comment:
Is this just a copy-paste error? I wouldn't expect this to be the case since FlutterEmbedderInit is a wrapper around GLFW initialization.